controllers/featureFlags.js

```diff
@@ -82,9 +84,33 @@ const deleteFeatureFlag = async (req, res) => {
   }
 }
 
+/**
+ * Get featureFlag config.
+ *
+ * This returns a list of feature flags and whether they are enabled or not.
+ * Feature flag being enabled depends on rollout config.
+ */
+const getConfig = async (req, res) => {
+  try {
+    const featureFlags = await featureFlagQuery.fetchFeatureFlag(false)
```

Comment:
Should the function be plural? `fetchFeatureFlags`, like the LHS variable name?

Reply:
Yes

Reply:
Done